.travis.yml

```diff
@@ -0,0 +1,17 @@
+matrix:
+  include:
+    - stage: lint
+      name: 'Scraper'
+      language: python
+      python:
+        - '3.6.7'
+      cache:
+        pip: true
+      install:
+        - pip install pipenv anybadge
+        - pipenv install --ignore-pipfile 
+      script:
+        - find . -iname "*.py" | xargs pylint --rcfile=.pylintrc --output-format=text > pylint.out || python -c "import sys;sys.exit($? in [1,32])" | cat pylint.out
+        - score=$(sed -n 's/^Your code has been rated at \([-0-9.]*\)\/.*/\1/p' pylint.out)
```

Comment:
We should be okay with say 8 score but not disable warnings for the code author & reviewer.

Reply:
set the lint stage to check for this. now pipeline will only fail for linting when the score is less than 8.